Automattic/jetpack
# Update banners that upsell Jetpack products

### Changes proposed in this Pull Request

This PR updates the banners that upsell Jetpack products and plans, according to the new offer.

Fixes: 1169247016322522-as-1187641228151000

### Implementation notes

- Updated the icons to match Calypso
- Refactored `PlanIcon` to use these icons
- Created a map to link features and product/plan upsells, so that subsequent updates are easy to make

_TODO: update jetpack.com redirects in WPcom_

### Testing instructions

- Select a self-hosted Jetpack site with a Free plan
- Run the plugin locally
- Check that each banner in the table below has been updated (copy, colour, and icon)
- In _Settings > Security > Backups and security scanning_, check that you see a different banner with a Security Daily plan


### Screenshots

<img width="1792" alt="Screen Shot 2020-09-22 at 4 41 39 PM" src="https://user-images.githubusercontent.com/1620183/93935429-99bff200-fcf2-11ea-88b3-8aec3f2f55fe.png">

### Does this pull request change what data or activity we track or use?
No

### Proposed changelog entry for your changes:
Update banners that upsell Jetpack plans and products to reflect the new offer
